### PR TITLE
use missed opportunity to detect number

### PR DESCRIPTION
**Pull request by igor-shtein**
add 'ringing' state number detection on new asterisks versions

### CODE REVIEW

**Comment by alexis-via** [comment]
Code review, no test (Asterisk 1.4 is soooo old, how could I test that :)
:+1:

File: asterisk_click2dial/asterisk_click2dial.py [hunk 1]
```diff
@@ -253,6 +253,13 @@ def _get_calling_number(self, cr, uid, context=None):
                     _logger.debug("Found a matching Event in 'Ring' state")
                     calling_party_number = chan.get('CallerIDNum')
                     break
+                # Compatibility with Asterisk 1.4
+                if (
```
Comment (pedrobaeza) [inline asterisk_click2dial/asterisk_click2dial.py:257]: Why this strange line break?

Comment (igor-shtein) [inline asterisk_click2dial/asterisk_click2dial.py:257]: I actually copy-paste "if" block from below, with comment and line breaks I assume.
I left comment about version coz it seems to be same api.
I`m running version 11.7 of asterisk and this change need to be in order to detect number in ringing state.
About line break, I used github built-in editor. I sure can reedit this with normal editor.

**Comment by pedrobaeza** [comment]
This can be merged although old.

**Pull request merged**